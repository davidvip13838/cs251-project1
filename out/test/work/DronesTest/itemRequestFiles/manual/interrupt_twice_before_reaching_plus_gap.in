4 5
3 7 13 34
