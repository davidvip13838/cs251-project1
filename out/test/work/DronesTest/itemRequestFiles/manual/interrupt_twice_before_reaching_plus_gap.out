1 13
2 23
0 33
3 44
